face_alignment/api.py

```diff
@@ -127,23 +132,28 @@ def get_landmarks_from_image(self, image_or_path, detected_faces=None, return_bb
 
         landmarks = []
         landmarks_scores = []
-        for i, d in enumerate(detected_faces):
+        for i, d in enumerate(detected_faces): 
             center = torch.tensor(
                 [d[2] - (d[2] - d[0]) / 2.0, d[3] - (d[3] - d[1]) / 2.0])
-            center[1] = center[1] - (d[3] - d[1]) * 0.12
-            scale = (d[2] - d[0] + d[3] - d[1]) / self.face_detector.reference_scale
+            if self.face_detector:
+                center[1] = center[1] - (d[3] - d[1]) * 0.12
+                scale = (d[2] - d[0] + d[3] - d[1]) / self.face_detector.reference_scale
+            else:
+                scale = (d[2]-d[0])/200

```

Comment:
can we add a comment explaining why 200? like "everywhere the scale is multiplied by 200, we don't know why, but if we divide it here by 200 it works"